### DISCUSSION TITLE
r.cost error: Ignoring point out of the window. No start points

### DISCUSSION THREAD

**Discussion by lucasbonald** [OSGeo/grass#3165] [Q&A] [answered]
Hello everyone! 
I'm facing some issues when I try to run r.cost on Grass 8.3. Whenever I run the function I get the message "warning: ignoring point out of the window. No Start_points". My data and project were in UTM first and then I convert all to WGS 84 Geographic, but the error persists. Can anyone help me with this?

This is the entire message
(Tue Jun 27 15:17:15 2023)                                                      
r.cost input=slope_wgs@PERMANENT output=cost_cat outdir=dit_cat start_coordinates=-37.335234235104664,-8.280350008588302 stop_coordinates=-37.13320625603865,-8.626278618357489
WARNING: Aviso, ignorando ponto fora da janela: -37.3352, -8.28035
ERROR: Sem pontos de partida
(Tue Jun 27 15:17:15 2023) Command ended with non-zero return code 1 (0 segundos)

- Operating System: Windows 10 Pro 21H2 and also Ubuntu 22.04 (I tried in both systems and I got the same error)
- GRASS GIS version 8.3.0 (Windows) and 8.2.2 (Ubuntu)

**Comment by neteler** [comment]
Please also post the output of `g.region -p`.

**Comment by lucasbonald** [comment]
(Tue Jun 27 15:37:04 2023)                                                      
g.region -p                                                                     
projection: 3 (Latitude-Longitude)
zone:       0
datum:      wgs84
ellipsoid:  wgs84
north:      1N
south:      0
west:       0
east:       1E
nsres:      1
ewres:      1
rows:       1
cols:       1
cells:      1
(Tue Jun 27 15:37:05 2023) Comando terminado (0 segundos)

**Comment by neteler** [comment, marked as answer]
Yeah, your https://grasswiki.osgeo.org/wiki/Computational_region is currently located in the Atlantic ocean :-)

First set it to the map you want to compute the cost surface on:

`g.region raster=slope_wgs -p`

and then `r.cost` which will respect the settings for the calculation region.

**Comment by lucasbonald** [comment]
Thank you very much for your help! It is my first time working with Grass itself.
